<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>42</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Youtube Downloader</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>by Nicklor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your URL : </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Download Video</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download Audio</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>660</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo.qrc">:/img/youtube_play1600.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="SubmitButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="thumbnail_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="author_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="views_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rating_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>220</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ã—</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="bestresbtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose best resolution</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="chooseresbtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose video to download : </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="bestresbtn_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose best resolution</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="chooseresbtn_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose audio to download : </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="conv_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Merge / Convert downloads</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Export format :</string>
     </property>
    </widget>
    <widget class="QLabel" name="export_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>checkBox_3</zorder>
    <zorder>export_label</zorder>
    <zorder>toolButton</zorder>
   </widget>
   <widget class="QPushButton" name="cancelDownloadButton">
    <property name="geometry">
     <rect>
      <x>594</x>
      <y>0</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel Download</string>
    </property>
   </widget>
   <zorder>listWidget</zorder>
   <zorder>label_3</zorder>
   <zorder>checkBox</zorder>
   <zorder>listWidget_2</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>SubmitButton</zorder>
   <zorder>thumbnail_label</zorder>
   <zorder>progressBar</zorder>
   <zorder>delButton</zorder>
   <zorder>clearButton</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>conv_widget</zorder>
   <zorder>cancelDownloadButton</zorder>
   <zorder>commandLinkButton</zorder>
   <zorder>rating_label</zorder>
   <zorder>views_label</zorder>
   <zorder>date_label</zorder>
   <zorder>author_label</zorder>
   <zorder>title_label</zorder>
   <zorder>label_6</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuCheck_If_File_Already_Exists">
     <property name="title">
      <string>Check If File Already Exists</string>
     </property>
     <addaction name="actionAsk_what_to_do"/>
     <addaction name="actionReplace_file"/>
     <addaction name="actionSkip_download"/>
    </widget>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionSystem"/>
     <addaction name="actionFusion"/>
     <addaction name="actionDark_Fusion"/>
     <addaction name="actionDark_Orange"/>
     <addaction name="actionOld_Windows"/>
     <addaction name="actionLight_Material"/>
     <addaction name="actionDark_Material"/>
    </widget>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
    <addaction name="actionChange_Downloads_Location"/>
    <addaction name="menuCheck_If_File_Already_Exists"/>
    <addaction name="menuTheme"/>
    <addaction name="actionInstall_extension_compatibility"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionChange_Downloads_Location">
   <property name="text">
    <string>Change Downloads Location ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAsk_what_to_do">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ask what to do</string>
   </property>
  </action>
  <action name="actionReplace_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Replace file</string>
   </property>
  </action>
  <action name="actionSkip_download">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Skip download</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSystem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>System</string>
   </property>
  </action>
  <action name="actionFusion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fusion</string>
   </property>
  </action>
  <action name="actionDark_Fusion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Fusion</string>
   </property>
  </action>
  <action name="actionDark_Orange">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Orange</string>
   </property>
  </action>
  <action name="actionOld_Windows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Old Windows</string>
   </property>
  </action>
  <action name="actionLight_Material">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light Material</string>
   </property>
  </action>
  <action name="actionDark_Material">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Material</string>
   </property>
  </action>
  <action name="actionInstall_extension_compatibility">
   <property name="text">
    <string>Install extension compatibility</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
